netscaler/config.go

```diff
@@ -41,8 +41,9 @@ func (c *NitroClient) AddResource(resourceType string, name string, resourceStru
 
 	resourceJSON, err := JSONMarshal(nsResource)
 
-	log.Printf("[TRACE] go-nitro: Resourcejson is " + string(resourceJSON))
-
+	if resourceType != "systemfile" {
```

Comment:
can resourceType be mixed case?
```suggestion
	if strings.ToLower(resourceType) != "systemfile" {
```

Reply:
or 
```suggestion
	if strings.EqualFold(resourceType, "systemfile") {
```